core-bundle/assets/controllers/tabs-controller.js

```diff
@@ -65,8 +70,8 @@ export default class TabsController extends Controller {
 
     panelTargetDisconnected(panel) {
         // Remove controls
-        const li = document.getElementById(panel.getAttribute('aria-labelledby')).parentElement;
-        li.remove();
+        const li = document.getElementById(panel.getAttribute('aria-labelledby'))?.parentElement;
+        li?.remove();
```

Comment:
```suggestion
        document.getElementById(panel.getAttribute('aria-labelledby'))?.parentElement?.remove();
```

IMHO the constant is not used for anything else.

Reply:
I had it in there, so that one can understand we are targeting the `li` element there. But I agree it's not really necessary. (Assets should still be the same after this change btw.)